server.go

```diff
@@ -834,3 +916,169 @@ func (w *response) ConnectionState() *tls.ConnectionState {
 	}
 	return nil
 }
+
+type quicConn struct {
```

Comment:
why is internal house keeping needed? quic-go doesn't do this for us? Or there is no other way to read/write for the correct stream?

Reply:
This was the only way I could wrap a `quic.Connection` and it's streams so I could place it in `srv.conns[]`.

Reply:
check